openslides_backend/action/actions/meeting/base_set_mediafile_action.py

```diff
@@ -49,7 +50,18 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 f"Invalid mimetype: {mediafile.get('mimetype')}, allowed are {self.allowed_mimetypes}"
             )
 
-        instance[self.field] = {instance.pop("place"): instance.pop("mediafile_id")}
+        place = instance.pop("place")
+        place_field = Meeting().get_field(self.field)
+        allowed_places = (
+            place_field.replacement_enum
+            if isinstance(place_field, BaseTemplateField)
+            else None
+        )
+        if allowed_places and place not in allowed_places:
+            raise ActionException(
+                f"Invalid place: {place}, allowed are {allowed_places}"
+            )
```

Comment:
I *think* the relation handling already does this check. Did you try it out without this implementation?